src/ring_theory/ideal/basic.lean

```diff
@@ -544,7 +544,9 @@ begin
 end
 
 /-- quotient by maximal ideal is a field. def rather than instance, since users will have
-computable inverses in some applications -/
+computable inverses in some applications.
+See note [reducible non-instances]. -/
+@[reducible]
 protected noncomputable def field (I : ideal α) [hI : I.is_maximal] : field I.quotient :=
```

Comment:
Why can't this one just be an instance?

Reply:
It has a docstring (somewhat) explaining it.

Reply:
I wonder whether this actually is a problem in practice. I imagine that all examples where the inverse is computable are hidden behind a new definition anyway. So then this instance wouldn't be found.